src/api/contact-form.ts

```diff
@@ -0,0 +1,101 @@
+import { WebClient, LogLevel } from '@slack/web-api';
+
+const TOKEN = process.env.SLACK_BOT_SY_TOKEN;
+
+/**
+ * Easily build this block structure via https://app.slack.com/block-kit-builder
+ */
+const createSlackMessage = ({ name, email, message }) => {
+  return [
+    {
+      type: 'header',
+      text: {
+        type: 'plain_text',
+        text: 'New form submission',
+      },
+    },
+    {
+      type: 'section',
+      text: {
+        type: 'mrkdwn',
+        text: 'We have received a new message.',
+      },
+    },
+    {
+      type: 'section',
+      fields: [
+        {
+          type: 'mrkdwn',
+          text: `*Name*:\n ${name}`,
+        },
+        {
+          type: 'mrkdwn',
+          text: `*E-mail*:\n ${email}`,
+        },
+      ],
+    },
+    {
+      type: 'divider',
+    },
+    {
+      type: 'section',
+      fields: [
+        {
+          type: 'plain_text',
+          text: message,
+        },
+      ],
+    },
+    {
+      type: 'context',
+      elements: [
+        {
+          type: 'mrkdwn',
+          text: 'Sent via _Satellytes Contact Form_',
+        },
+      ],
+    },
+  ];
+};
+
+export default async function handler(req, res) {
```

Comment:
You can declare the `req` and `res` argument with the Gatsby types to make it simpler to work with:

```
(req: GatsbyFunctionRequest, res: GatsbyFunctionResponse)
```

(not sure about the correct import here in Github, but you will figure it out ;-))

Reply:
I bet I copied this from some example but that's a good idea indeed.

Reply:
I fixed that 👍